docs/content/how-tos/rules/FL0074.md

```diff
@@ -0,0 +1,32 @@
+---
+title: FL0074
+category: how-to
+hide_menu: true
+---
+
+# FavourConsistentThis (FL0074)
+
+*Introduced in `0.20.3`*
+
+## Cause
+
+Rule to detect inconsistent usage of `this`.
+
+## Rationale
+
+Using `this` and `self` interchangeably within the same code-base affects readability.
+
+## How To Fix
+
+Consistently use the symbol specified in the configuration (either `this` or `self`).
+
+## Rule Settings
+
+    {
+        "favourConsistentThis": {
+			"enabled": true,
+			"config": {
+				"symbol": "this"
+			}
+		}
```

Comment:
indentation of this json fragment seems to be wrong, at least compared to other docs of other rules

Reply:
I think this could be because I used tabs instead of spaces. will fix. thanks